webapp/stores/post_store.jsx

```diff
@@ -259,22 +260,32 @@ class PostStoreClass extends EventEmitter {
         this.postsInfo[id].postList = combinedPosts;
     }
 
+    // updates/adds all the instance of "post" in every postsInfo
     storePost(post, isNewPost = false) {
```

Comment:
Just for information:

Until now this method used to update/add the post only in its channel. The problem was that in the permalink view the post is linked to the post being viewed, meaning postsInfo contains not only channel ids but also posts ids. With this PR, storePost will go through all the postsInfo that contains the post being stored and update/add it.